### PR TITLE
Correct formatting

### PR DESCRIPTION
**Pull request by ataggart**
See: https://circleci.com/gh/cloverage/cloverage/89

### CODE REVIEW

**Comment by ataggart** [comment]
Not sure if it is intentionally disabled, but CircleCI could be configured to build pull request branches.

File: cloverage/test/cloverage/instrument_test.clj [hunk 1]
```diff
@@ -75,7 +75,7 @@
     (t/is (= 1 (both-defn 1)))))
 
 (t/deftest test-exclude-calls
-  (let [form    '(inc 100) 
+  (let [form   '(inc 100)
```
Comment (lvh) [inline cloverage/test/cloverage/instrument_test.clj]: FWIW, I'm definitely with you here; this was totally correct the first time round. Sorry (I inherited the config and this project lacks someone with the resources to maintain it properly :/)

Comment (ataggart) [reply cloverage/test/cloverage/instrument_test.clj]: D'oh, hold on.

Comment (ataggart) [reply cloverage/test/cloverage/instrument_test.clj]: Ok, it was just the trailing whitespace for that line. Amended the commit to just address that.

**Pull request merged**